### PR TITLE
getLiquidationMarginUsd should take a sizeDelta and calc mm/im from that

### PR DESCRIPTION
**Pull request by 0xjocke**

### CODE REVIEW

File: contracts/interfaces/ILiquidationModule.sol [hunk 1]
```diff
@@ -74,11 +74,13 @@ interface ILiquidationModule is IBasePerpMarket {
     function isMarginLiquidatable(uint128 accountId, uint128 marketId) external view returns (bool);
 
     /**
-     * @notice Returns the IM (initial maintenance) and MM (maintenance margin) for a given account and market.
+     * @notice Returns the IM (initial maintenance) and MM (maintenance margin) for a given account, market and size.
```
Comment (davidvuong) [inline contracts/interfaces/ILiquidationModule.sol:77]:
```suggestion
     * @notice Returns the IM (initial maintenance) and MM (maintenance margin) for a given account, market, and sizeDelta.
```

File: test/integration/modules/LiquidationModule.test.ts [hunk 2]
```diff
@@ -2263,4 +2263,187 @@ describe('LiquidationModule', () => {
...
+      const { marketId } = await depositMargin(bs, genTrader(bs));
+      const invalidAccountId = 42069;
+      await assertRevert(
+        PerpMarketProxy.getLiquidationMarginUsd(invalidAccountId, marketId, 0),
+        `AccountNotFound("${invalidAccountId}")`,
+        PerpMarketProxy
+      );
+    });
+
+    it('should revert when invalid accountId', async () => {
```
Comment (davidvuong) [inline test/integration/modules/LiquidationModule.test.ts:2289]: Duplicated test?

File: contracts/interfaces/ILiquidationModule.sol [hunk 3]
```diff
@@ -74,11 +74,13 @@ interface ILiquidationModule is IBasePerpMarket {
...
+     * If you want IM/MM for an existing postions you should pass 0 for sizeDelta.
```
Comment (davidvuong) [inline contracts/interfaces/ILiquidationModule.sol:78]:
```suggestion
     * Specify a sizeDelta of 0 for the current IM/MM of an existing position.
```

**Review by davidvuong — APPROVED** [review]
Minor comments and dup test otherwise lgtm

**Comment by 0xjocke** [comment]
#71 replaced by

**Pull request closed without merging**